tests/UnifiedSpecTests/Util.php

```diff
@@ -88,6 +88,8 @@ final class Util
             'createChangeStream' => ['pipeline', 'session', 'fullDocument', 'fullDocumentBeforeChange', 'resumeAfter', 'startAfter', 'startAtOperationTime', 'batchSize', 'collation', 'maxAwaitTimeMS', 'comment', 'showExpandedEvents'],
             'createFindCursor' => ['filter', 'session', 'allowDiskUse', 'allowPartialResults', 'batchSize', 'collation', 'comment', 'cursorType', 'hint', 'limit', 'max', 'maxAwaitTimeMS', 'maxScan', 'maxTimeMS', 'min', 'modifiers', 'noCursorTimeout', 'oplogReplay', 'projection', 'returnKey', 'showRecordId', 'skip', 'snapshot', 'sort'],
             'createIndex' => ['keys', 'comment', 'commitQuorum', 'maxTimeMS', 'name', 'session', 'unique'],
+            'createSearchIndex' => ['model'],
+            'createSearchIndexes' => ['models'],
```

Comment:
Since this is probably your first interaction with this array, I'm curious if you have any thoughts on it. It dates back to https://github.com/mongodb/mongo-php-library/commit/2a81aafc96beada8cc58a18381817b34c1497395 for [PHPLIB-662](https://jira.mongodb.org/browse/PHPLIB-662) but I'm not entirely convinced it's more useful than annoying.

Reply:
IMO the schema validation of does not cover which fields are allowed for each operation. But this should be validated in the spec repository, not here.

Reply:
[Incorporate referenced entity operations into the schema version](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#incorporate-referenced-entity-operations-into-the-schema-version) in the unified test format spec explains that in more detail. The problem with validating operations in the schema is that we'd have to bump schema versions much more frequently.